docs/releases/2.17.md

```diff
@@ -29,3 +29,9 @@ Here are other changes related to the redesign:
 
 ## Upgrade considerations
 
+### Hallo legacy rich text editor has moved to an external package
+
+* Hallo was deprecated in in [Wagtail v2.0 (February 2018)](https://docs.wagtail.org/en/stable/releases/2.0.html#new-rich-text-editor) and has not received any bug fixes with only a minimal level of support since then.
```

Comment:
```suggestion
* Hallo was deprecated in [Wagtail v2.0 (February 2018)](https://docs.wagtail.org/en/stable/releases/2.0.html#new-rich-text-editor) and has had only a minimal level of support since then.
```